bundle/compliance/lib/common/common.rego

```diff
@@ -77,6 +77,16 @@ duration_gte(duration, min_duration) {
 	true
 }
 
+# checks that the date given is not earlier than the provided duration
+# date: time in nanoseconds
+date_less_than(date, duration) {
```

Comment:
`date_less_than` - "checks that the date given is **not** earlier than the provided duration" docs and naming seem inconsistent, no?

Reply:
Changed to `The function determines whether the given date occurs earlier than the specified duration from the current time`.

still fells sloppy though.